### ISSUE TITLE
Video SQL Joins: Phase 3

### ISSUE THREAD

**Issue by HagayHaut** [learn-co-curriculum/phase-3-lecture-videos-sql-joins#5]
### Canvas Link

https://learning.flatironschool.com/courses/5180/pages/video-sql-joins?module_item_id=398156

### Concern

Ix's content is great, as always. But video quality is pretty poor, and not being able to read the screen makes it very hard to follow along

### Suggested Changes

Zooming in on video or re-uploading in higher quality would be very helpful! Love the content as always Ix!

**Comment by lizbur10** [comment]
Hi @HagayHaut ! Thanks for the feedback. I have made a note of the problem so it will be addressed when the video is next updated.

Thanks for contributing to the Flatiron School curriculum 💙
